src/components/InputField/index.css

```diff
@@ -0,0 +1,30 @@
+.input-block{
+  position: relative;
+}
+.input-field{
+  padding: 16px 16px 16px 60px;
+  border: none;
+  background: #fff;
+  color: grey; /* dark-grey */
```

Comment:
why is need comment?

Reply:
@AlexButov 
There should be var(--dart-grey), but this variable is in the other commit, so that's temporary value. I will fix it in next commit